src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,16 @@
 
 public class MainApp {
 
+    public void main(String[] args) {
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
+        }
+    }
+    private Machine Truck;
+    private Machine Bulldozer;
+    private Machine Excavator;
+    Machine[] machines = new Machine[]{Truck, Bulldozer, Excavator};
```

Comment:
this is wrong approach

you need to create machines array directly in the main method. having these objects as fields is redundant

what if you would need 5 machines instead of 3? you would have to declare 5 fields. and what if 100? :)